containers/ecr-viewer/src/app/api/services/dialects/sqlserver.ts

```diff
@@ -0,0 +1,53 @@
+import { MssqlDialect } from "kysely";
+import { ConnectionPool } from "mssql";
+import * as tarn from "tarn";
+import * as tedious from "tedious";
+
+const dbConfig = ConnectionPool.parseConnectionString(
+  process.env.DATABASE_URL || "",
+);
+
+export const dialect = {
+  dialect: new MssqlDialect({
+    tarn: {
+      ...tarn,
+      options: {
+        min: dbConfig.pool.min || 0,
+        max: dbConfig.pool.max || 100,
+      },
+    },
+    tedious: {
+      ...tedious,
+      connectionFactory: () => {
+        try {
+          const res = new tedious.Connection({
+            authentication: {
+              options: {
+                password: dbConfig.password || process.env.SQL_SERVER_PASSWORD,
+                userName: dbConfig.user || process.env.SQL_SERVER_USER || "sa",
+                domain: dbConfig.domain,
+              },
+              type: dbConfig.authentication?.type || "default",
+            },
+            options: {
+              database: dbConfig.database || "master",
```

Comment:
I think we accidentally missed [crypto credentials](https://github.com/CDCgov/dibbs-ecr-viewer/blob/0172fbfdbf91f3d13c3542093a0b114a7bd9b474/containers/ecr-viewer/src/app/data/db/sqlserver_db.ts#L18C1-L21C1). 
```
      cryptoCredentialsDetails: {
        ciphers: process.env.DB_CIPHER,
      },
```